tests/ui/manual_abs_diff.rs

```diff
@@ -1,5 +1,5 @@
+#![allow(clippy::manual_checked_sub)]
```

Comment:
Why is this needed? You should not detect situations like

```rust
if a <= b { b - a } else { a - b }
```

in your lint, or at least you should make sure that it only triggers if `manual_abs_diff` doesn't.